airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -365,6 +365,20 @@ def generate_key(record):
         )
 
 
+class InventoryItems(ChildSubstream):
+
+    parent_stream_class: object = Products
+    cursor_field = "updated_at"
```

Comment:
This property is not needed here, since the `ChildSubstream` contains the `cursor_field` property already.

Reply:
removed